app/code/Chandan/Orderlisticon/Block/Order/History.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Chandan\Orderlisticon\Block\Order;
+
+use \Magento\Framework\App\ObjectManager;
+use \Magento\Sales\Model\ResourceModel\Order\CollectionFactoryInterface;
+
+/**
+ * Sales order history block
```

Comment:
Please describe why you override

Reply:
Instead of changing the core file, we can use the override method because with a new concept of dependency injection where classes inject dependencies (different objects) for an object instead of that object manually creating them internally. That way overriding and manipulating with classes is much easier and allows us more ways of extending the native functionalities.
There are three type 
   Preference 
   Event and Observer 
   Plugin